src/Application/GameStarter.cpp

```diff
@@ -131,29 +132,23 @@ GameStarter::GameStarter(GameStarterOptions options): _options(std::move(options
     _application->component<GameWindowHandler>()->UpdateWindowFromConfig(_config.get());
 
     // Init renderer.
-    _renderer = RendererFactory().createRenderer(_options.headless ? RENDERER_NULL : _config->graphics.Renderer.value(), _config);
+    auto rendererType = _options.headless ? RENDERER_NULL : _config->graphics.Renderer.value();
```

Comment:
We are in the almost-always-not-auto crowd. The type name here isn't something insane like `std::vector<int>::const_iterator`, let's spell it out.